source/extensions/all_extensions.bzl

```diff
@@ -1,7 +1,7 @@
 load("@bazel_skylib//lib:dicts.bzl", "dicts")
 load("@envoy_build_config//:extensions_build_config.bzl", "DISABLED_BY_DEFAULT_EXTENSIONS", "EXTENSIONS")
 
-GLOBAL_DENYLIST = [k for k, v in DISABLED_BY_DEFAULT_EXTENSIONS.items()]
+GLOBAL_DENYLIST = [k for k in DISABLED_BY_DEFAULT_EXTENSIONS.keys()]
```

Comment:
I mean you can just write: `GLOBAL_DENYLIST = DISABLED_BY_DEFAULT_EXTENSIONS.keys()` :)

Reply:
Done :)